ecs/variables.tf

```diff
@@ -81,6 +85,10 @@ variable "task_role_arn" {
   default = ""
 }
 
+variable "name" {
```

Comment:
Might be worth making this a little more descriptive and/or adding a description

Reply:
I've added a description in.